integration/apply_test.go

```diff
@@ -112,3 +112,23 @@ func TestApplyStatusCheckFailure(t *testing.T) {
 		})
 	}
 }
+
+func TestApplyMultipleNamespaces(t *testing.T) {
+	testutil.Run(t, "TestApplyMultipleNamespaces", func(t *testutil.T) {
+		MarkIntegrationTest(t.T, NeedsGcp)
+		ns, client := SetupNamespace(t.T)
+		ns2, _ := SetupNamespace(t.T)
+
+		_, _, fErr := replaceInFile("namespace1", ns.Name, fmt.Sprintf("%s/skaffold.yaml", "testdata/helm-multi-namespaces"))
```

Comment:
nit: might make sense to use a separate constant for the test data directory path instead of repeating "testdata/helm-multi-namespaces". Might make it easier to update the path if necessary in the future.